README.MD

```diff
@@ -104,13 +132,13 @@ Steps:
 E.g.
 
 ```bash
```

Comment:
```suggestion
```console
```
And then we'd keep the dollar signs as prompts. Sounds good?

Reply:
Why the dollar signs? The idea of these snippets is to lower the barrier to people wanting to contribute, these commands are always the same and just need to be copy pasted (which can't happen that easily if you have to copy paste line by line removing the dollar sign)..

Reply:
The dollar signs indicate something to be typed at a prompt (which in Bash usually defaults to something ends in `$ `). Is there a way to use CSS `:before` or similar to produce the dollar signs?

Reply:
Not here, as we don't have access to such CSS rules in GitHub Markdown rendered.
Also, the idea of those snippets is to let people quickly copy-paste these commands. We don't have stuff that doesn't need to be typed at a prompt, everything is supposed to.

Reply:
Does this mean [Google developer documentation style guide article "Documenting command-line syntax" section "Command prompt"](https://developers.google.com/style/code-syntax#prompt) is a bad example?

> If your command-line instructions show multiple lines of input, then start each line of input with the `$` prompt symbol.
> [...]
> 👍 **Recommended:**  
> Enter the following code into the terminal:
> 
>     $ adb devices

Reply:
> If your command-line instructions show **multiple** lines of input
We only have 3 lines total, and each input is one line only, with clear context and no need to clarify it (we don't use monospaced blocks for anything else).
That is only a guideline. In fact, if you keep reading the very same document, they don't use any `$` at  all.